docs/how-to/containerd-kata.md

```diff
@@ -104,17 +104,17 @@ $ popd
 
 ### Configure containerd to use Kata Containers
 
-By default, the configuration of containerd is located at `/etc/containerd/config.toml`, and the 
+By default, the configuration of containerd is located at `/etc/containerd/config.toml`, and the
 `cri` plugins are placed in the following section:
 
 ```toml
 [plugins]
-  [plugins.cri]
-    [plugins.cri.containerd]
-      [plugins.cri.containerd.default_runtime]
+  [plugins."io.containerd.grpc.v1.cri"]
+    [plugins."io.containerd.grpc.v1.cri".containerd]
+      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
         #runtime_type = "io.containerd.runtime.v1.linux"
 
-    [plugins.cri.cni]
+    [plugins."io.containerd.grpc.v1.cri".cni]
```

Comment:
So, it depends a lot on the version of containerd being used.

AFAIR, there are still supported versions of containerd out there that will only accept the old configuration. :-/

Reply:
It'd be worth adding a note about:
"If using config `version = 1`, use ${what_we_have}, otherwise (`version = 2`) use ${what_you_have_proposed}".

Reply:
@fidencio Good idea. I'll do that.

Reply:
@fidencio Done in the latest push.